webplugin/js/app/kommunicate-ui.js

```diff
@@ -348,6 +308,29 @@ KommunicateUI={
         }
     });
 },
+searchFaqs:function(data){
+KommunicateKB.searchFaqs({
+    data:
+        { appId: data.appId, query: document.getElementById("km-faq-search-input").value }
+    , success: function (response) {
+        if (response.data && response.data.length === 0 && $applozic(".km-no-results-found-container").hasClass("n-vis")) {
+            $applozic(".km-no-results-found-container").addClass("vis").removeClass("n-vis");
```

Comment:
please refrain from using jquery and use **modifyClasslist** function in KommunicateCommons